### PR TITLE
Wizard only onboarding

### LINKED ISSUE #55777: Remove manual setup snippets from Next, Remix and Sveltekit getting started pages
### Problem Statement

Currently, we show both, wizard and manual setup in the getting started docs. This is potentially confusing for users as they might think they need to add additional code as to what the wizard already set up.

### Solution Brainstorm

Remove the manual usage code snippets (plus the dynamic Sentry feature button on top since they're no longer useful) 

for:

* [ ] NextJS
* [ ] Remix
* [ ] SvelteKit

Instead, we only link to the manual setup page as an alternative to using the wizard.

### Product Area

Projects - Project Creation

### PR DESCRIPTION
**Pull request by smeubank**
~Remix~, SvelteKit, and Nextjs

Someone can validate if these changes, by someone I mean a real developer :D

SvelteKit: local
image

NextJs: local
image

~Remix: local~
image

closes #55777

EDIT: Remix is still open, waiting on  https://github.com/getsentry/sentry-wizard/issues/433, merging this without remix. Remix change is tracked in #54615

### CODE REVIEW

File: static/app/gettingStartedDocs/javascript/remix.spec.tsx [hunk 1]
```diff
@@ -1,21 +1,12 @@
 import {render, screen} from 'sentry-test/reactTestingLibrary';
 
 import {StepTitle} from 'sentry/components/onboarding/gettingStartedDoc/step';
-import {ProductSolution} from 'sentry/components/onboarding/productSelection';
 
-import {GettingStartedWithRemix, nextSteps, steps} from './remix';
+import {GettingStartedWithSvelteKit, steps} from './sveltekit';
```
Comment (ArthurKnaus) [inline static/app/gettingStartedDocs/javascript/remix.spec.tsx:5]: Copy-🍝
```suggestion
import {GettingStartedWithRemix, steps} from './remix';
```

**Review by ArthurKnaus — COMMENTED** [review]
Code itself looks good 👍

**Review by Lms24 — CHANGES_REQUESTED** [review]
Only requesting changes so we don't merge this until https://github.com/getsentry/sentry-wizard/issues/433 is resolved. Let's only publish this once the wizard is fully ready.

**Comment by smeubank** [comment]
> Only requesting changes so we don't merge this until [getsentry/sentry-wizard#433](https://github.com/getsentry/sentry-wizard/issues/433) is resolved. Let's only publish this once the wizard is fully ready.

@Lms24 good to go?

cc @ArthurKnaus

**Review by Lms24 — APPROVED** [review]
LGTM, good call on proceeding here  w/o remix

**Comment by Lms24** [comment]
@smeubank fixed the linter errors, should be good now 🤞

**Pull request merged**